### DISCUSSION TITLE
Can we assume no newlines in station names?

### DISCUSSION THREAD

**Discussion by parkertimmins** [gunnarmorling/1brc#311] [Rule clarifications] [answered]
The FAQ says that station names can be arbitrary UTF-8 with the only restriction that they do not contain semicolons. This implies that a station name could contain a newline. On the other hand, the name "billion **row** challenge" seems to implies a single row/newline per entry.

**Comment by iziamos** [comment]
I like what you're thinking :)

**Comment by gunnarmorling** [comment, marked as answer]
Good question. Yes, you can assume that there are no new line characters in station names. PR welcome for clarifying it in the README.

**Comment by parkertimmins** [comment]
Great, thanks! (and thanks for making this challenge!) I went ahead and pushed a README change in https://github.com/gunnarmorling/1brc/pull/317

**Reply by gunnarmorling** [reply]
Excellent, thx!
